todoman/cli.py

```diff
@@ -294,7 +296,13 @@ def move(ctx, list, ids):
 @click.option('--reverse/--no-reverse', default=True,
               help='Sort tasks in reverse order (see --sort). '
               'Defaults to true.')
-def list(ctx, lists, all, urgent, location, category, grep, sort, reverse):
+@click.option('--due', default=None, help='Only show tasks due in DUE hours',
+              type=int)
+# TODO: we might want a `porcelain` flag here to print this is a
```

Comment:
JSON seems like the obvious candidate. There are CLI tools dedicated to selecting fields from JSON, for scripting purposes.